spec/integration_spec.rb

```diff
@@ -823,6 +823,11 @@ class SerializedObject < ActiveRecord::Base
     expect(facets.first['highlighted']).to eq('<em>bl</em>a')
     expect(facets.first['count']).to eq(1)
   end
+
+  it "should reindex after settings change" do
+    Color.index.set_settings!(:searchableAttributes => [:short_name])
+    Color.reindex
+  end
```

Comment:
I would recommend we add some `expect` here, this test is not validating much in its current form.

Reply:
as we're trying to asses that it does not fail, I'll change it to `expect(Color.reindex)`. WDYT?

Reply:
It depends, was it ultimately failing raising a Ruby exception? In this case you should rather to `expect { Color.reindex }.not_to raise_error`.

Reply:
👍 gotcha!